src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequestHelper.cs

```diff
@@ -40,5 +42,32 @@ internal static async Task<MsalTokenResponse> RefreshAccessTokenAsync(MsalRefres
 
             return dict;
         }
+
+        internal static void ProcessFetchInBackgroundAsync(Func<Task<AuthenticationResult>> fetchAction, ICoreLogger logger)
+        {
+            _ = Task.Run(async () =>
+            {
+                try
+                {
+                    await fetchAction().ConfigureAwait(false);
+                }
+                catch (MsalServiceException ex)
+                {
+                    string logMsg = $"Background fetch failed. Is AAD down? { ex.IsAadUnavailable()}. MsalServiceException {ex.ToString()}";

```

Comment:
The `ex.ToString()` does not seem in alignment with the [PII guidance](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging#logging-in-msal) from the docs.

*Important: exception messages are not captured if enablePiiLogging is set to false (PII = personal identifiable info). If you need to contact support, please try to send Verbose logs with PII . Do not post PII logs on GitHub!*

Reply:
This is a good point. @SameerK-MSFT, please use `void ErrorPiiWithPrefix(Exception exWithPii, string prefix);` which will take care of hiding the PII message if needed.